src/WebJobs.Script/Utility.cs

```diff
@@ -466,6 +470,16 @@ internal static bool ShouldInitiliazeLanguageWorkers(IEnumerable<FunctionMetadat
             return ContainsNonDotNetFunctions(functionsListWithoutProxies);
         }
 
+        internal static string GetCurrentLanguageRuntime(IEnumerable<FunctionMetadata> functions)
```

Comment:
This logic seems suspect. Don't we require all functions to be of the same language? I thought we checked that elsewhere. Why are we treating DotNet specially?

Reply:
For dot net logic is different from the rest of the languages. csx, .dll map to dotnet language but function metadata shows language as csharp